clis/common/src/cli.rs

```diff
@@ -757,13 +774,13 @@ impl Cli {
                 stun_server.parse::<IpAddr>().expect("Invalid IPv4 address"),
                 stun_port,
             )) {
-                Ok(data) => {
-                    cli_res!(res; (i"Public Address: {:?}", data.public_ip));
-                    cli_res!(res; (i"Nat Type: {:?}", data.nat_type));
+                Ok(_data) => {
+                    // cli_res!(res; (i"Public Address: {:?}", data.public_ip));
```

Comment:
why is this commented out?

Reply:
My bad, leftover code from debugging issues. The missing whitespace between i and the string literal was giving me weird compiler errors before. I'll bring it back.